src/components/intro/intro.jsx

```diff
@@ -75,6 +72,12 @@ class Intro extends React.Component {
                                 </div>
                             )
                         }
+                        
+                        <Video
+                            className={classnames('intro-videos', !this.state.videoOpen && 'invisible')}
```

Comment:
Do we need the `invisible` class if we don't render the video component when `this.state.videoOpen` is false?

Reply:
I removed the style, it was left behind from another previous solution that I tried